device/arista/x86_64-arista_7170_64c/Arista-7170-64C/buffers_defaults_t0.j2

```diff
@@ -10,37 +14,70 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
-            "size": "33329088",
+            "size": "{{ ingress_lossless_pool_size }}",
             "type": "ingress",
-            "mode": "dynamic",
-            "xoff": "7827456"
+            "mode": "dynamic"
         },
-        "egress_lossy_pool": {
-            "size": "26663272",
-            "type": "egress",
+        "ingress_lossy_pool": {
+            "size": "{{ ingress_lossy_pool_size }}",
+            "type": "ingress",
             "mode": "dynamic"
         },
         "egress_lossless_pool": {
-            "size": "42349632",
+            "size": "{{ egress_lossless_pool_size }}",
             "type": "egress",
-            "mode": "static"
+            "mode": "dynamic"
+        },
+        "egress_lossy_pool": {
+            "size": "{{ egress_lossy_pool_size }}",
+            "type": "egress",
+            "mode": "dynamic"
         }
     },
     "BUFFER_PROFILE": {
-        "ingress_lossy_profile": {
+        "ingress_lossless_profile": {
             "pool":"[BUFFER_POOL|ingress_lossless_pool]",
-            "size":"0",
-            "static_th":"11075584"
+            "size":"4096",
```

Comment:
why do we change it so small?

Reply:
ingress lossless profile should have no limit.

Reply:
When threshold mode dynamic is configured, the above size value is ignored by the barefoot SDK. Buffers are carved out dynamically for the ingress buffers. Same comment applies for egress. So this size value should be ok.

Reply:
the size here is the reserved buffer for the pg, for lossless traffic, it should be zero.

Reply:
@zzhiyuan can you change size to 0 and give it a shot? I believe that should work with the SDE. You can reach out to me if you face any issues changing this value to 0

Reply:
I've changed all the lossless buffer sizes to 0. Thanks